src/pybritive/britive_cli.py

```diff
@@ -201,7 +220,7 @@ def _is_saml_user(token):
             ).get('username', '')
 
             return username.startswith('SAML')
```

Comment:
this is likely changing with Britive Internal Directory. Let's see what updates need to occur.